<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>centralTabs</class>
 <widget class="QTabWidget" name="centralTabs">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>286</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabsClosable">
   <bool>false</bool>
  </property>
  <property name="movable">
   <bool>true</bool>
  </property>
  <property name="tabBarAutoHide">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="demoPage">
   <attribute name="title">
    <string>Demo Utilities</string>
   </attribute>
   <layout class="QHBoxLayout" name="demoPageLayout">
    <item>
     <layout class="QVBoxLayout" name="stepsLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="scriptsPage">
   <attribute name="title">
    <string>Script Studio</string>
   </attribute>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>74</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="scriptsLayout"/>
   </widget>
  </widget>
  <widget class="QWidget" name="productionPage">
   <attribute name="title">
    <string>Production</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
